changelog/1478.feature

```diff
@@ -0,0 +1,4 @@
+Added `--show-capture` feature. You can choose 'no', 'stdout' or 'stderr' option.
```

Comment:
We appreciate trying to write a long descriptive message here, but unfortunately `towncrier` (which we use to manage the changelog) does not allow for multi-paragraph entries.

I think this feature can be summarized thus:

```rst
New ``--show-capture`` command-line option that allows to specify how to display captured output when tests fail: ``no``, ``stdout``, ``stderr`` or ``both`` (the default).
```

Which is short and to the point.

Besides that, I think we also need to mention this on the docs, I believe `capture.rst` is the appropriate place for it. Just add a new paragraph to the `Default stdout/stderr/stdin capturing behaviour` section like this:

```rst
Default stdout/stderr/stdin capturing behaviour
---------------------------------------------------------

During test execution any output sent to ``stdout`` and ``stderr`` is
captured.  If a test fails its captured
output will be shown along with the failure traceback (this behavior 
can be configured by the ``--show-capture`` command-line option). 
```

Reply:
@feuillemorte just realized that this bit should be added to the docs, would you mind doing it? Thanks!

Reply:
@nicoddemus I found print() =) removed...